userweb/app/styles/main.scss

```diff
@@ -261,6 +261,36 @@ table {
     }
 }
 
+/**
+ * QR Code
+ */
+#qrcode {
+    position: relative;
+}
+
+#qrcode div {
+    position: absolute;
+    color: rgba(0,0,0,.8);
+    font-size: 10pt;
+    font-style: italic;
+    top: -9999px;
+}
+
+#qrcode qr:hover + div {
+    top: -50px !important;
+    left: 150px;
+    z-index: 999;
+    border: 6px solid rgba(0,0,0,.15);
+    border-radius: 6px;
+    padding: 6px;
+    background-color: rgba(255,255,255,.95);
+    -webkit-box-shadow: 0 0 6px rgba(132, 132, 132, .75);
+    -moz-box-shadow: 0 0 6px rgba(132, 132, 132, .75);
+    box-shadow: 0 0 6px rgba(132, 132, 132, .75);
+    -webkit-border-radius: 4px;
+    -moz-border-radius: 4px;
+}
+
```

Comment:
After testing this, the overlayed image flickers a bit when the cursor is over.

We have replaced this CSS to a simple _resize_ behaviour. Please, give it a try if you have the time.

```/**
 * QR Code
 */
#qrcode {
    position: relative;
}

#qrcode:hover div {
    display:block!important;
}

#qrcode:hover > qr {
    display:none;
}

#qrcode div {
    display: none;
    color: rgba(0,0,0,.8);
    font-size: 10pt;
    font-style: italic;
}
```

What do you think?